protocol/revapi.json

```diff
@@ -73,6 +73,18 @@
           "old": "method io.camunda.zeebe.protocol.record.value.ImmutableProcessInstanceModificationRecordValue io.camunda.zeebe.protocol.record.value.ImmutableProcessInstanceModificationRecordValue::withActivatedElementInstanceKeys(java.lang.Iterable<? extends java.lang.Long>)",
           "justification": "This method had a wrong name. It was only used internally as it provides no value for others."
         },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "io.camunda.zeebe.protocol.record.intent.IncidentIntent::shouldBlacklistInstanceOnError()",
+          "justification": "This method had a racial loaded term. It was replaced with a better name, this shouldn't been an issue as it was for internally use only."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "io.camunda.zeebe.protocol.record.intent.JobIntent::shouldBlacklistInstanceOnError()",
+          "justification": "This method had a racial loaded term. It was replaced with a better name, this shouldn't been an issue as it was for internally use only."
+        },
```

Comment:
❓ Why do we need these twice? They are also on line 206-215, except here it references the return type. There's only 1 method in the intents.

Reply:
Ups this was a left over sorry.